deps/chakrashim/lib/chakra_shim.js

```diff
@@ -428,7 +429,7 @@
   var microTasks = [];
 
   function patchUtils(utils) {
-    var isUintRegex = /^(0|[1-9]\\d*)$/;
+    var isUintRegex = /^(0|[1-9]\d*)$/;
```

Comment:
No, because we don't want to match numbers starting with 0 like "012", "00", etc.